app/javascript/dashboard/components/widgets/conversation/linear/LinkIssue.vue

```diff
@@ -123,7 +130,11 @@ const linkIssue = async () => {
     issues.value = [];
     onClose();
   } catch (error) {
-    useAlert(t('INTEGRATION_SETTINGS.LINEAR.LINK.LINK_ERROR'));
+    const errorData = error.response.data;
+    const errorMessage =
+      errorData?.error?.errors?.[0]?.message ||
+      t('INTEGRATION_SETTINGS.LINEAR.LINK.LINK_ERROR');
+    useAlert(errorMessage);
```

Comment:
Please take a look at this [comment](https://github.com/chatwoot/chatwoot/pull/9538#discussion_r1616654267).